cmd/uhc/account/users/cmd.go

```diff
@@ -151,8 +151,8 @@ func run(cmd *cobra.Command, argv []string) {
 
 }
 
-// getRolesFromUser gets all roles a specific user possesses.
-func getRolesFromUser(account *amv1.Account, conn *client.Connection) []string {
+// GetRolesFromUser gets all roles a specific user possesses.
+func GetRolesFromUser(account *amv1.Account, conn *client.Connection) []string {
```

Comment:
Please move the function into a package (or make a new one) and use it for both the `status` & `users` commands.

Reply:
Agreed, this is best for the project. I just didn't want to scope creep SREP-1860. It's good to go now unless you'd prefer the new package be named differently.